pandas/tests/groupby/test_counting.py

```diff
@@ -195,3 +195,14 @@ def test_ngroup_respects_groupby_order(self):
                                 g.ngroup())
             assert_series_equal(Series(df['group_index'].values),
                                 g.cumcount())
+
+    def test_count_with_datetime(self):
```

Comment:
can u parametrize to count over tz naive (and aware);
also over Timedeltas and Period would be great

Reply:
change test name to _datetimelike